components/web/apps/org.wso2.carbon.apimgt.store/src/main/fragments/application-details/public/js/tabs.js

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/**
+ * Javascript to enable link to tab. Change hash for page-reload, If hashed page name is available load that page
+ * If no hash value is present load the default page which is overview-tab
+ */
+function loadFromHash() {
+    var hash = document.location.hash;
```

Comment:
location.hash is not safe for IE, recommend to use window.location

Reply:
Thanks for the review. Since we have discontinued UUF in favour of React.js, this file is no longer available in this repository.